examples/convert_ieeg_to_bids.py

```diff
@@ -194,39 +164,23 @@
     shutil.rmtree(bids_root)
 
 # %%
-# Now we just need to specify a few iEEG details to make things work:
-# We need the basename of the dataset. In addition, :func:`write_raw_bids`
-# requires the ``.filenames`` attribute of the Raw object to be non-empty,
-# so since we
-# initialized the dataset from an array, we need to do a hack where we
-# temporarily save the data to disc before reading it back in.
+# Now we just need make a :class:`mne_bids.BIDSPath` to save the data.
+#
+# .. warning:: By passing ``acpc_aligned=True``, we are affirming that
+#              the T1 in this dataset is aligned to ACPC. This is very
+#              difficult to check with a computer which is why this
+#              step is required.
 
 # Now convert our data to be in a new BIDS dataset.
-bids_path = BIDSPath(subject=subject_id,
-                     task=task, acquisition="ecog", root=bids_root)
+bids_path = BIDSPath(subject=subject_id, task=task, root=bids_root)
 
 # write `raw` to BIDS and anonymize it into BrainVision format
 write_raw_bids(raw, bids_path, anonymize=dict(daysback=30000),
-               overwrite=True)
+               acpc_aligned=True, overwrite=True)
```

Comment:
If `acpc_aligned=True`, then do we need to `remove_fiducials` outside of `write_raw_bids`? Could we then do it inside the write function and just document this behavior?

Reply:
Might be better just to pass the montage as just mentioned in the previous comment but without that, no that wouldn't be possible because you have to remove before assigning to raw.